src/main/java/seedu/finance/logic/commands/SpendCommand.java

```diff
@@ -28,7 +28,7 @@
             + "[" + PREFIX_CATEGORY + "CATEGORY]...\n"
             + "Example: " + COMMAND_WORD + " "
             + PREFIX_NAME + "John Doe "
-            + PREFIX_AMOUNT + "123 "
+            + PREFIX_AMOUNT + "$123.23 "
```

Comment:
Should not have `$` as it is not intuitive for user to use `PREFIX_AMOUNT` (`$/`).